WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -500,6 +504,14 @@ class MySiteViewModel @Inject constructor(
         )
     }
 
+    private fun enableQuickLinkRibbonFocusPoints(): Boolean {
+        return defaultABExperimentTab == MySiteTabType.DASHBOARD
+    }
```

Comment:
1. Since we're not enabling the focus points here but just checking if they should be enabled, we can rename the function to reflect this.
2. Prefer using an expression body for functions with the body consisting of a single expression.

```suggestion
    private fun shouldEnableQuickLinkRibbonFocusPoints() = defaultABExperimentTab == MySiteTabType.DASHBOARD
```

P.S. Same applies to `enableSiteMenuFocusPoints()` function.

Reply:
@ashiagr Thanks for this suggestion. I completely agree with your suggestion. I have done the changes in the commit [ef5c2dd](https://github.com/wordpress-mobile/WordPress-Android/pull/16361/commits/ef5c2dd8ba00f91b11d496a01a4adff35d04a618).  Also I felt that the function name `shouldEnableSiteItemsFocusPoints` is better than `shouldEnableSiteMenuFocusPoints`. Wdyt?. 🤔

Reply:
I too like `shouldEnableSiteItemsFocusPoints()`. 🙂

Reply:
Verified in 2d22c65, ef5c2dd